libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.kt

```diff
@@ -1032,7 +1033,7 @@ open class OAuthWebviewHelper : KeyChainAliasCallback {
             if (id?.screenLockTimeout?.compareTo(0) == 1) {
                 SalesforceSDKManager.getInstance().registerUsedAppFeature(FEATURE_SCREEN_LOCK)
                 val timeoutInMills = (id?.screenLockTimeout ?: 0) * 1000 * 60
-                instance.screenLockManager?.storeMobilePolicy(
+                (instance.screenLockManager as ScreenLockManager?)?.storeMobilePolicy(
```

Comment:
Off topic, but I really hate `instance` here.  If there is anything worse than the Singleton Pattern it is making references to Singletons 😞